### ISSUE TITLE
Overriding pre-loaded value from v-model

### ISSUE THREAD

**Issue by gabrielboliveira** [niksmr/vue-masked-input#3]
When I have a pre-loaded value on the model, it is changed by the component. Example:

```js
...
    data () {
        return {
            time: '00:00:00:00'
        }
    },
...
```
```html
<masked-input v-model="time" mask="+ 11:11" />
```

It doesn't load the value from the `time` model, instead it overrides with empty value. The only way I made it work was to reset the time variable on `mounted()` method:

```js
mounted () {
    this.time = '00:00:00:00'
}
```

**Comment by niksmr** [comment]
Fixed in 0.3.1. Please update

**Comment by gabrielboliveira** [comment]
Fixed! Thanks
